server/src/main/java/org/elasticsearch/index/get/ShardGetService.java

```diff
@@ -253,15 +249,11 @@ private GetResult innerGetLoadFromStoredFields(
             if (fetchSourceContext.fetchSource() == false) {
                 source = null;
             } else if (fetchSourceContext.includes().length > 0 || fetchSourceContext.excludes().length > 0) {
-                Map<String, Object> sourceAsMap;
                 // TODO: The source might be parsed and available in the sourceLookup but that one uses unordered maps so different.
                 // Do we care?
```

Comment:
@nik9000 @romseygeek is this TODO still valid here? It seems like the code around it changed quite a bit.

Reply:
`SourceLookup` in `ShardGetSevice` has been removed in #20158, so this todo is invalid now. 
I open #85921 to remove another similar todo in this file, please help to reivew it.